docs/definitions.md

```diff
@@ -1,293 +0,0 @@
-
-#### format (string)
-
-`format` defines if the function is running or not in `hot function` mode.
-
-To define the function execution as `hot function` you set it as one of the following formats:
-
-- `"http"`
-
-### 'Hot function' Only Properties
-
-This properties are only used if the function is in `hot function` mode
-
-
-## Calls and their statuses
-
-### Sync/Async Call statuses
-
-With each function call, no matter would that be sync or async server makes a record of this it.
-While execution async function server returns `call_id`:
- 
-```json
- {
-    "call_id": "f5621e8b-725a-4ba9-8323-b8cdc02ce37e"
- }
-```
-that can be used to track call status using following command:
- 
-```sh
- 
- curl -v -X GET ${FN_API_URL}/v1/calls/f5621e8b-725a-4ba9-8323-b8cdc02ce37
- 
-```
-
-```json
-{
-    "message": "Successfully loaded call",
-    "call": {
-        "id": "f5621e8b-725a-4ba9-8323-b8cdc02ce37e",
-        "status": "success",
-        "completed_at": "2017-06-02T15:31:30.887+03:00",
-        "created_at": "2017-06-02T15:31:30.597+03:00",
-        "started_at": "2017-06-02T15:31:30.597+03:00",
-        "app_name": "newapp",
-        "path": "/envsync"
-    }
-}
-
-```
-
-Server response contains timestamps(created, started, completed) and execution status for this call.
-
-For sync call `call_id` can be retrieved from HTTP headers:
-```sh
-curl -v localhost:8080/r/newapp/envsync 
-*   Trying ::1...
-* TCP_NODELAY set
-* Connected to localhost (::1) port 8080 (#0)
-> GET /r/newapp/envsync HTTP/1.1
-> Host: localhost:8080
-> User-Agent: curl/7.51.0
-> Accept: */*
-> 
-< HTTP/1.1 200 OK
-< Fn-Call-Id: f5621e8b-725a-4ba9-8323-b8cdc02ce37e
-< Date: Fri, 02 Jun 2017 12:31:30 GMT
-< Content-Length: 489
-< Content-Type: text/plain; charset=utf-8
-< 
-...
-```
-Corresponding HTTP header is `Fn-Call-Id`.
-
-### Per-route calls
-
-In order get list of per-route calls please use following command:
-
-```sh
-curl -X GET ${FN_API_URL}/v1/app/{app}/calls/{route}
-
-```
-Server will replay with following JSON response:
-
-```json
-{
-    "message": "Successfully listed calls",
-    "calls": [
-        {
-            "id": "80b12325-4c0c-5fc1-b7d3-dccf234b48fc",
-            "status": "success",
-            "completed_at": "2017-06-02T15:31:22.976+03:00",
-            "created_at": "2017-06-02T15:31:22.691+03:00",
-            "started_at": "2017-06-02T15:31:22.691+03:00",
-            "app_name": "newapp",
-            "path": "/envsync"
-        },
-        {
-            "id": "beec888b-3868-59e3-878d-281f6b6f0cbc",
-            "status": "success",
-            "completed_at": "2017-06-02T15:31:30.887+03:00",
-            "created_at": "2017-06-02T15:31:30.597+03:00",
-            "started_at": "2017-06-02T15:31:30.597+03:00",
-            "app_name": "newapp",
-            "path": "/envsync"
-        }
-    ]
-}
-```
-
-### Pagination
```

Comment:
just double checking we defined 'how pagination works' in the new docs somewhere? it's slightly non-standard (it's not an actual 'page number' and some users may need details to make sense of it). the rest of this file seems covered elsewhere